### PR TITLE
TECH-214: Initialize Express.js backend 

### PR DESCRIPTION
**Pull request by dborowiecki**
Ticket: [TECH-214](https://govstack-global.atlassian.net/browse/TECH-214)

Changes: 
1. Added subdirectory with express.js backend application 
2. Extended docker-compose by mongoDB and backend app services. 

[TECH-214]: https://govstack-global.atlassian.net/browse/TECH-214?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,4 +46,21 @@ docker-compose up --build
 or 
 ```shell
  docker-compose up
-```
\ No newline at end of file
+```
+
+## Backend 
+Testing application has a simple backend written using express.js. 
+A more detailed description of the backend's part can be found in `/server/readme.MD`.
```
Comment (KarolinaKopacz) [inline README.md:53]:
```suggestion
A more detailed description of the backend's part can be found in `/server/README.md`. 
```

File: server/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Testing app backend 
+
+Express.js server provides access to report results. 
+
+---
+## Requirements
+- Node.js
+- Yarn
```
Comment (KarolinaKopacz) [inline server/README.md:8]: What version?

File: server/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Env variables that regex matches `API_KEY_*` pattern are used to determine lists of API keys authorized to push report results to the storage. API keys have to be passed as an `x-api-key` header in incoming requests.
+
+### Other 
+* `PORT`
+
+By default, application runs on port `5000`.
+
+## Running the project
+
+    $ yarn dev
```
Comment (KarolinaKopacz) [inline server/README.md:42]:
```suggestion
$ yarn dev
```

File: server/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Uses `nodemon` for hot reload.
+
+## Simple build for production
+
+    $ yarn build
```
Comment (KarolinaKopacz) [inline server/README.md:48]:
```suggestion
$ yarn build
```

File: server/README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+
```
Comment (KarolinaKopacz) [inline server/README.md:50]:
```suggestion
```

**Review by KarolinaKopacz — CHANGES_REQUESTED** [review]

**Review by dszafranek — APPROVED** [review]

File: server/README.md (hunk 3 above)
Comment (dborowiecki) [reply server/README.md:42]: Indent is intentional for separate block for highlighting the block of code: 

    $ yarn dev

it has same effect as wrapping text with ``` block.

File: README.md (hunk 1 above)
Comment (dborowiecki) [reply README.md]: fixed

File: server/README.md (hunk 2 above)
Comment (dborowiecki) [reply server/README.md]: Added

File: server/README.md (hunk 4 above)
Comment (dborowiecki) [reply server/README.md:48]: It's same as for yarn dev comment, please find my response

File: server/README.md (hunk 5 above)
Comment (dborowiecki) [reply server/README.md]: Removed

**Pull request merged**